cosi/service/cosi_test.go

```diff
@@ -34,6 +34,33 @@ func TestServiceCosi(t *testing.T) {
 		msg, res.Signature))
 }
 
+func TestServiceCosi2(t *testing.T) {
+	defer log.AfterTest(t)
+	log.TestOutput(testing.Verbose(), 4)
+	local := onet.NewTCPTest()
+	// generate 5 hosts, they don't connect, they process messages, and they
+	// don't register the tree or entitylist
+	hosts, el, _ := local.GenTree(5, false)
+	defer local.CloseAll()
+
+	// Send a request to the service to the second host
+	client := NewTestClient(local)
+	msg := []byte("hello cosi service")
+	serviceReq := &SignatureRequest{
+		Roster:  el,
+		Message: msg,
+	}
+	dst := el.List[1]
```

Comment:
Can you do a `for _, dst := range el.List` and then test for all hosts? I think this would be the best test we can do for that part.